query-engine/connectors/sql-connector/src/database/connector_transaction.rs

```diff
@@ -0,0 +1,51 @@
+use prisma_query::connector::Queryable;
+
+use connector_interface::*;
```

Comment:
I think the interface is explicitly designed to allow for wild card imports. I tried your suggestion though. It did not compile because `RecordFinder`, `NestedDisconnect`.. were missing. Since we will a lot of more functions the import on the connector interface would explode anyway.